### DISCUSSION TITLE
Release Candidate available?

### DISCUSSION THREAD

**Discussion by joeaudette** [albar965/littlenavmap#1032] [General]
I've been waiting patiently for the 2.8.11 release. I've seen some commits referencing rc2, wondering if it is possible to get access to that or not, or is there a private group that tests the release candidates?

**Comment by albar965** [comment]
Sure. Here is the latest 2.8.11.rc3 : https://magentacloud.de/s/oy26WafZMwxr6y5
Included changelog and user manual PDF are updated but not the online user manual.
Let me know if you find any issues. I did a few big changes (more internally) which have the potential to produce bugs. :upside_down_face: 

Alex

**Reply by joeaudette** [reply]
Awesome! Many thanks! Will test it later today.

**Comment by joeaudette** [comment]
I am so stupid. I didn't even realize there was a dropdown under grouped queries until this release. I had never seen all those other queries.
